tests/unit/controllers/books-test.js

```diff
@@ -46,20 +46,18 @@ module('Unit | Controller | books', function(hooks) {
   });
 
   test('saveBook', function(assert) {
+    assert.expect(3);
+
     const { book, controller } = this;
+
+    book.isValid = true;
     controller.send('saveBook', book);
```

Comment:
`controller.send` is from the days when the action resided in the route. Let's clean this up.

```js
controller.saveBook(book);
```

Reply:
I stand [corrected](https://github.com/zoltan-nz/library-app/pull/73/files#r392718888).